src/lib/EEPROM/elrs_eeprom.cpp

```diff
@@ -1,38 +1,51 @@
 #include "elrs_eeprom.h"
+#include <Arduino.h>
 
 #ifdef PLATFORM_STM32
-extEEPROM EEPROM(kbits_2, 1, 1);
+    extEEPROM EEPROM(kbits_2, 1, 1, 0x51);
```

Comment:
IIRC the r9mm uses the default address 0x50.  Something to keep in mind when reusing this lib for the receivers.